services/movie.py

```diff
@@ -0,0 +1,40 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,

```

Comment:
It also accepts None then it is mean Optional[list[int]] 
Check also in other places